input/number/reports.md

```diff
@@ -0,0 +1,28 @@
+## This page should be used for casual reporting of issues.
+
+For help on formatting, see: 
+- [How to put multiple lines in a Markdown table cell (multiline table)](https://alvinalexander.com/text/how-put-multiple-lines-markdown-table-cell-multiline-table)
+- [Markdown table generator](https://www.tablesgenerator.com/markdown_tables)
+
+### <a name="valid-issues">Valid issue types:</a> 
+- Inconsistency with other similar input elements
+- Non-extensibly (functionality)
+- Non-extensibly (styling)
+- Lacks built-in accessibility 
```

Comment:
This is a good example of something that would need to be dug into as I actually don't believe this is true. This may however be true on certain OSes or certain browsers which is why doing a current state dig into the input is valuable

Reply:
Right, so my idea for a report is that it isn't necessarily true. It's just someone saying something. It would need further investigation were we/you to recognise it as an issue.